ui/lib/ldap/addon/components/page/library/details/accounts.hbs

```diff
@@ -1,39 +1,41 @@
-<div class="has-top-margin-l is-flex-align-start">
```

Comment:
The diff is hard to read but I just changed the classes on the top level div to grid and then wrapper the `Hds::Card::Container` in a div. Other than that it's the same.

Reply:
yay for grids! 🙌